VoxelSniperCore/src/main/java/com/github/kevindagame/brush/multiBlock/GenerateTreeBrush.java

```diff
@@ -26,6 +28,7 @@ public class GenerateTreeBrush extends AbstractBrush {
     // Tree Variables.
     private final Random random = new Random();
     private final ArrayList<IBlock> branchBlocks = new ArrayList<>();
+    private final HashMap<String, Boolean> leafBlocks = new HashMap<>();
```

Comment:
Why is this a map instead of a set or list? Is lookup faster for a map?

Reply:
A Hashmap Has a near o(1) loopup performance compared to an ArrayList's o(n). Which on >10k iterations makes a difference. I've changed it to a hashSet though which is the same but only 1 value

Reply:
yeah, I know of the array/arraylist, but wasn't sure on the set